proto/kvrpcpb.proto

```diff
@@ -1009,3 +1009,11 @@ message CheckLeaderResponse {
     repeated uint64 regions = 1;
     uint64 ts = 2;
 }
+
+// Additional flags for a write batch. They should be set in the `flags` field in `RaftRequestHeader`.
+enum WriteBatchFlag {
```

Comment:
Not necessary to define these flags in proto.

Reply:
OK. I decide to put it directly inside TiKV.